apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -188,6 +188,7 @@ const configuredUiItems = new Map<string, UiItem>([
     {
       initialize: async () => PropertyGridManager.initialize(),
       createUiItemsProviders: () => [
+        // eslint-disable-next-line deprecation/deprecation
         new PropertyGridUiItemsProvider({
```

Comment:
Can you update it to use the new create widget function?

Reply:
changed